doc/auth.md

```diff
@@ -300,7 +300,87 @@ Check out the following user guides for examples of protecting services with Kua
 * One HTTPRoute can only be targeted by one AuthPolicy.
 * One Gateway can only be targeted by one AuthPolicy.
 * AuthPolicies can only target HTTPRoutes/Gateways defined within the same namespace of the AuthPolicy.
-* 2+ AuthPolicies cannot target network resources that define/inherit the same exact hotname.
+* 2+ AuthPolicies cannot target network resources that define/inherit the same exact hostname.
+
+#### Limitation: Multiple network resources with identical hostnames
+
+Kuadrant currently does not support multiple AuthPolicies simultaneously targeting network resources that declare identical hostnames. This includes multiple HTTPRoutes that specify the same hostnames in the `spec.hostnames` field, as well as HTTPRoutes that specify a hostname that is identical to a hostname specified in a listener of one of the route's parent gateways or HTTPRoutes that don't specify any hostname at all thus inheriting the hostnames from the parent gateways. In any of these cases, **a maximum of one AuthPolicy targeting any of those resources that specify identical hostnames is allowed**.
+
+Moreover, having **multiple resources that declare identical hostnames** may lead to unexpected behavior and therefore **should be avoided**.
+
+This limitation is rooted at the underlying components configured by Kuadrant for the implementation of its polcies and the lack of information in the data plane regarding the exact route that honored by the API gateway in cases of conflicting hostnames.
```

Comment:
typo: polcies -> policies
I am not 100% sure about that "...exact route that honored by...". Should not it be "...that is honored by..."